packages/extension/lib/util.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  getCookieUrl: (cookie = {}) => {
-    const prefix = cookie.secure ? 'https://' : 'http://'
```

Comment:
moved into the automation cookie code in the server as it is the only place its used